js/src/setup-mc/setup-stepper/setup-free-listings/shipping-time/shipping-time-setup/countries-time-input-form/index.js

```diff
@@ -16,7 +16,7 @@ const CountriesTimeInputForm = ( props ) => {
 
 	useEffect( () => {
 		setValue( savedValue );
-	}, [ savedValue ] );
+	}, [ savedValue.time ] );
```

Comment:
🚧 Same thing, React ESLint will complain about this: `"React Hook useEffect has a missing dependency: 'savedValue'. Either include it or remove the dependency array.eslintreact-hooks/exhaustive-deps"`.

Reply:
Not judging, just curious!

Why the GH linter is not failing? In theory the GH actions and the linter task should help us to handle this if it's a blocker.

Reply:
There's the `--quiet` option in [this workflow](https://github.com/woocommerce/google-listings-and-ads/blob/1.9.0/.github/workflows/js-css-linting.yml#L35) to skip the JS linter warnings. Otherwise, it would show 14 lint warnings by GH annotations in every PR. We could remove the `--quiet` if it's more helpful than the current.

Reply:
Is not there a way to promote `"React Hook useEffect has a missing dependency...` as error? Since it's a blocker issue.

Reply:
> Is not there a way to promote "React Hook useEffect has a missing dependency... as error?

It can be done by changing 'warn' to 'error' here:
https://github.com/woocommerce/google-listings-and-ads/blob/cf03ab591744d709d599fbba77cdf6d9a19b2bd5/.eslintrc.js#L14-L15